doc/source/release.rst

```diff
@@ -37,10 +37,57 @@ analysis / manipulation tool available in any language.
 * Binary installers on PyPI: https://pypi.org/project/pandas
 * Documentation: http://pandas.pydata.org
 
+pandas 0.23.1
+-------------
+
+**Release date**: June 12, 2018
+
+This is a minor release from 0.23.0 and includes a number of bug fixes and
```

Comment:
just a note for the message you will use for announcing, I would explicitly mention that some regression have been fixed